### PR TITLE
Feat/Guide knobs and variations

### PR DESCRIPTION
**Pull request by Jamerrone**
- [Trello](https://trello.com/c/Dc5hrOgE/88-5-add-variations-and-knobs-to-guide)
- [Branch Demo](https://deploy-preview-99--leanwebkit.netlify.com/guide/)

### CODE REVIEW

File: src/client/components/app-core/app-core.stories.css [hunk 1]
```diff
@@ -34,3 +34,42 @@
...
+  display: flex;
+  margin-bottom: var(--spacing-default);
+}
+ .icons-demo p {
+  margin: 0;
+}
+ .icons-demo img {
+  margin-left: var(--spacing-default);
+}
+ .icons-demo .s-64 {
```
Comment (vbvnvmfjg) [inline src/client/components/app-core/app-core.stories.css]: I would remove the .icons-demo .s-xxx classes and style it inline instead.

File: src/client/components/app-core/app-core.stories.js [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+import { withReadme } from 'storybook-readme';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import readme from './readme.md';
 
-import './index.css'
-import './app-core.stories.css'
+import './index.css';
+import './app-core.stories.css';
+
+const iconSizes = [
```
Comment (vbvnvmfjg) [inline src/client/components/app-core/app-core.stories.js]: I would use a simpler array, like [64, 120, 144]

**Comment by Jamerrone** [comment]
@anandgraves @jbmoelker feedback processed

File: src/client/components/app-core/app-core.stories.js [hunk 3]
```diff
@@ -105,21 +112,30 @@ storiesOf('App core', module)
...
+        </demo>
+        <demo title="Primary" inset>
           <button class="button button--primary">Primary</button>
-          <code>class="button button--primary button--pending"</code>
           <button class="button button--primary button--pending">Pending</button>
         </demo>
+        <demo title="Try it out!" inset knobs>
+          <div>
+            <button class="button ${
+              boolean('isPrimary', false) ? 'button--primary ' : ''
```
Comment (vbvnvmfjg) [inline src/client/components/app-core/app-core.stories.js]: Could you change 'isPrimary' to 'Primary button'?

File: src/client/components/app-core/app-core.stories.js [hunk 4]
```diff
@@ -105,21 +112,30 @@ storiesOf('App core', module)
...
+            }
+            ${boolean('isPending', false) ? 'button--primary button--pending ' : ''}">
```
Comment (vbvnvmfjg) [inline src/client/components/app-core/app-core.stories.js]: Could you change 'isPending' to 'Pending button'?

**Review by vbvnvmfjg — APPROVED** [review]

**Pull request merged**